packages/@aws-cdk/aws-route53-targets/lib/load-balancer-target.ts

```diff
@@ -11,7 +12,7 @@ export class LoadBalancerTarget implements route53.IAliasRecordTarget {
   public bind(_record: route53.IRecordSet): route53.AliasRecordTargetConfig {
     return {
       hostedZoneId: this.loadBalancer.loadBalancerCanonicalHostedZoneId,
-      dnsName: this.loadBalancer.loadBalancerDnsName,
+      dnsName: cdk.Fn.join('.', ['dualstack', this.loadBalancer.loadBalancerDnsName]),
```

Comment:
Why not ``dualstack.${this.loadBalancer.loadBalancerDnsName}`` and let `core` handle the case when a `Fn.join` is effectively needed.

Reply:
even better, let's go with that!